lib/X25519KeyPair.js

```diff
@@ -29,15 +29,21 @@ export class X25519KeyPair extends LDKeyPair {
    * @param {string} options.controller
    * @param {string} options.id
    * @param {string} options.publicKeyBase58 - Base58 encoded Public Key
-   * unencoded is 32-bytes.
-   * @param {string} options.privateKeyBase58 - Base58 Private Key
-   * unencoded is 32-bytes.
+   *   (unencoded is 32-bytes).
+   * @param {string} [options.privateKeyBase58] - Base58 Private Key
+   *   (unencoded is 32-bytes).
```

Comment:
Is this *always* true or can it be 64? I realize that technically, the private key material itself is 32 bytes, but it is often shipped around as 64 bytes (having the public key material -- also 32 bytes -- concatenated onto the end). This is how the very commonly used sodium libraries represent the private key (I believe).

I'm happy for it to be consistent -- I just wanted to make sure it is always 32 bytes if the docs are going to say so.

Reply:
@dlongley I think I was going by inline comments in whatever lib we extracted this code from. Happy to just remove the part in parentheses.

Reply:
Removed.